src/test/java/gov/usgs/wma/waterdata/collections/ColectionsControllerIT.java

```diff
@@ -0,0 +1,76 @@
+package gov.usgs.wma.waterdata.collections;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
```

Comment:
These are junit4 constructs. please exclude junit from the dbunit dependency in the pom and convert these to junit5.